README.md

```diff
@@ -1,24 +1,77 @@
-# README
+# Hello-Rails-Back-End
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This is the backend side of a Hello-World web-app using Rails.
 
-Things you may want to cover:
+# Hello-React-Front-End
+[React Back-end app](https://github.com/Terbeche/hello-react-front-end/tree/dev)
 
-* Ruby version
+## Built With
```

Comment:
- [ ] Kindly update your README file actually you're missing a list of technologies used in this project, let's add it.

Reply:
well resolved! ✔️